src/components/Tabs/Tabs.tsx

```diff
@@ -1,72 +1,43 @@
 import * as React from "react";
 import {
+  chakra,
+  Box,
   Tabs as ChakraTabs,
   TabList,
   Tab,
   TabPanels,
   TabPanel,
-  // useStyleConfig,
-  chakra,
+  useMultiStyleConfig,
   useStyles,
   useTab,
 } from "@chakra-ui/react";
 
-import bem from "../../utils/bem";
 import generateUUID from "../../helpers/generateUUID";
-
-// TODO: later when styling
-// enum TabsType {
```

Comment:
Ended up not using variants which makes this simpler.

Reply:
Makes sense.  At this point, there are no true variations in the overall `Tabs` UI.